src/registry.rs

```diff
@@ -189,13 +189,25 @@ impl PortableRegistry {
             (id, ty)
         })
     }
+
+    /// Clears all docs from types.
+    ///
+    /// Use when the docs are not required to produce smaller encoded metadata.
+    pub fn clear_docs(&mut self) {
```

Comment:
Not for this PR, but maybe at some point it'd be useful to return only the docs, or a `HashMap<TypeName, Vec<&'static str>>`. For instance in `substrate-api-sidecar` some of our docs are automatically generated from the rustdocs by way of the metadata. Could be useful to get only the docs.

Reply:
Mentioned in https://github.com/paritytech/scale-info/issues/94